cmd/tapcli/assets.go

```diff
@@ -3,12 +3,11 @@ package main
 import (
 	"encoding/hex"
 	"fmt"
-	"io/ioutil"
-
 	"github.com/lightninglabs/taproot-assets/tapcfg"
 	"github.com/lightninglabs/taproot-assets/taprpc"
 	"github.com/lightninglabs/taproot-assets/taprpc/mintrpc"
 	"github.com/urfave/cli"
+	"os"
```

Comment:
I think `"os"` should be grouped with the entries before the `"github.com/..."` entries. Which is the pattern we use in the rest of the codebase. However, the linter would pass either way.

```
import (
	"encoding/hex"
	"fmt"
        "os"

	"github.com/lightninglabs/taproot-assets/tapcfg"
	"github.com/lightninglabs/taproot-assets/taprpc"
	"github.com/lightninglabs/taproot-assets/taprpc/mintrpc"
	"github.com/urfave/cli"
)
```